frontend/src/hooks/api/getters/useInstanceStats/useInstanceStats.ts

```diff
@@ -0,0 +1,58 @@
+import useSWR from 'swr';
+import { useMemo } from 'react';
+import { formatApiPath } from 'utils/formatPath';
+import handleErrorResponses from '../httpErrorResponseHandler';
+
+interface InstanceStats {
+    instanceId: string;
+    timestamp: Date;
+    versionOSS: string;
+    versionEnterprise?: string;
+    users: number;
+    featureToggles: number;
+    projects: number;
+    contextFields: number;
+    roles: number;
+    groups: number;
+    environments: number;
+    segments: number;
+    strategies: number;
+    SAMLenabled: boolean;
+    OIDCenabled: boolean;
+}
+
+export interface IInstanceStatsResponse {
+    stats?: InstanceStats;
+    refetchGroup: () => void;
+    loading: boolean;
+    error?: Error;
+}
+
+export const mapGroupUsers = (users: any[]) =>
+    users.map(user => ({
+        ...user.user,
```

Comment:
user.user reads a bit weird to me. Is there another word to explain what the parameter is?

Reply:
copy/paste 🤕